test/uk/gov/hmrc/transitmovements/controllers/ArrivalsControllerSpec.scala

```diff
@@ -360,4 +362,41 @@ class ArrivalsControllerSpec
     }
   }
 
+  "getArrivalMessages" - {
+
+    val request = FakeRequest("GET", routes.ArrivalsController.getArrivalMessages(eoriNumber, movementId).url)
+
+    "must return OK and a list of message ids" in {
```

Comment:
nitpick -- must return OK and a list of message ids if an arrival is found.

Reply:
done